events/views.py

```diff
@@ -4,13 +4,46 @@
 from django.urls import reverse
 from .models import Event, Location
 from django.contrib.auth.decorators import login_required
+from django.utils import timezone
+from .forms import EventFilterForm
 
 # Create your views here.
 
 
 def index(request):
-    events = Event.objects.filter(is_active=True)
-    return render(request, "events/events.html", {"events": events})
+    events = Event.objects.filter(end_time__gt=timezone.now(), is_active=True).order_by(
```

Comment:
timezone.now() will give u time based on IST
please add Ny time if u want u can refer my pr

Reply:
Adressed the concern, used pytz.